RELEASES.md

```diff
@@ -57,6 +58,110 @@ Get acquainted with the latest EDP releases.
 * [Version 2.6.0](#2.6.0)
 </details>
 
+## Version 3.10.0 <a name="3.10.0"></a> (October 11, 2024)
+
+## What's New
+
+We are enhancing the platform towards new code storage and Version Control Systems support. We are pleased to introduce Bitbucket Cloud as a new Version Control System in KubeRocketCI.
+
+Our platform now integrates with Large Language Models, such as ChatGPT, as an intelligent [chat assistant](https://docs.kuberocketci.io/docs/next/user-guide/add-ai-assistant). Depending on your specified settings, it can perform a variety of tasks. We can also set up this assistant for you. Please contact our team for more details.
+
+A new release brings a new look. Check out our new [documentation site](https://docs.kuberocketci.io/), which features improved navigation and documentation versioning support.
+
+We have updated terminology within the platform. Environments are now referred to as Deployment Flows, and Stages are now Environments. These changes align our terminology with globally recognized terms, reducing user confusion. For more details, please refer to our updated [Glossary](https://docs.kuberocketci.io/docs/next/glossary).
+
+We have added the ability to specify custom build and review pipelines with just one click when editing or creating new codebase branches. This enhancement increases flexibility when working with codebases, allowing for easier modifications to pipeline logic based on specific requirements.
+
+Another significant enhancement is the introduction of a new section called Pipelines. This section centralizes pipeline observability by consolidating all pipelines in one place. We are continuing to migrate Tekton functionality into our platform. Consequently, we have consolidated our pipelines into a single location. The new Pipelines section displays the entire Tekton Pipeline Run history, simplifying pipeline management and observability.
+
+Users can now deliver their applications in the form of RPM packages, implemented using Ansible. Correspondingly, support for C and C++ languages with make and CMake build tools has been added to the platform.
```

Comment:
let's split this and:

Users can now automate the deployment of their applications packaged as RPMs, utilizing Ansible for this process.